packages/editor-ui/src/components/TagsDropdown.vue

```diff
@@ -22,8 +22,23 @@
 			@visible-change="onVisibleChange"
 			@remove-tag="onRemoveTag"
 		>
+			<!-- key is id+index for keyboard navigation to work well with filter -->
+			<n8n-option
+				v-for="(tag, i) in options"
+				:key="tag.id + '_' + i"
+				ref="tagRefs"
+				:value="tag.id"
+				:label="tag.name"
+				class="tag"
+				data-test-id="tag"
+			/>
+
 			<n8n-option
-				v-if="options.length === 0 && filter && createEnabled"
+				v-if="
+					options.filter((tag: ITag) => tag && tag.name === filter).length === 0 &&
+					filter &&
+					createEnabled
```

Comment:
Can you please extract this condition into a computed property?

Reply:
Great idea. Done. :+1: